tachyon/zk/plonk/halo2/verifier.h

```diff
@@ -314,6 +316,71 @@ class Verifier : public VerifierBase<PCS> {
         F::template LinearCombination</*forward=*/true>(expressions, proof.y);
     return expected_h_eval /= (proof.x.Pow(this->pcs_.N()) - F::One());
   }
+
+  size_t GetSizeOfAdviceInstanceColumnQueries(
+      const ConstraintSystem<F>& constraint_system) {
+    const std::vector<AdviceQueryData>& advice_queries =
+        constraint_system.advice_queries();
+    size_t size = advice_queries.size();
+    if constexpr (PCS::kQueryInstance) {
+      const std::vector<InstanceQueryData>& instance_queries =
+          constraint_system.instance_queries();
+      size += instance_queries.size();
+    }
+    return size;
+  }
+
+  template <ColumnType C>
+  void CreateColumnQueries(const std::vector<QueryData<C>>& queries,
+                           const std::vector<Commitment>& commitments,
+                           const std::vector<F>& evals,
+                           const Proof<F, Commitment>& proof,
+                           std::vector<Opening>& openings,
+                           std::vector<F>& points) const {
+    for (size_t i = 0; i < queries.size(); ++i) {
+      const QueryData<C>& query = queries[i];
+      const ColumnKey<C>& column = query.column();
+      points.push_back(query.rotation().RotateOmega(this->domain(), proof.x));
+      openings.emplace_back(
+          base::DeepRef<const Commitment>(&commitments[column.index()]),
+          base::DeepRef<const F>(&points.back()), evals[i]);
+    }
+  }
+
+  bool DoVerify(
+      const std::vector<std::vector<Commitment>>& instance_commitments_vec,
+      const VerifyingKey<PCS>& vkey, const Proof<F, Commitment>& proof) {
+    std::vector<Opening> queries;
```

Comment:
How about renaming 'queries' to 'openings'? That's why, we don't use `VerifierQuery` any more. 
(It is just recommendation, not necessary)

Reply:
Yeap, I wanted so, too! But, at this moment, like `AdviceQuery`, we name all coupled with `XXXQuery`. So how about renaming all in the next iteration?